R/mod_body_coc_competition.R

```diff
@@ -0,0 +1,475 @@
+#' body_coc_competition UI Function
+#'
+#' @description A shiny Module.
+#'
+#' @param id,input,output,session Internal parameters for {shiny}.
+#'
+#' @noRd 
+#'
+#' @importFrom shiny NS tagList 
+mod_body_coc_competition_ui <- function(id){
+  ns <- NS(id)
+  tagList(
+    ui_picker_program(
+          inputId = ns("pe_provider"),
+          label = "Select your CoC-funded Provider",
+          choices = sort(pe_summary_validation()$AltProjectName) |> unique(),
+          selected = pe_summary_validation()$AltProjectName[1],
+          options = shinyWidgets::pickerOptions(liveSearch = TRUE,
+                                  liveSearchStyle = 'contains')
+        ),
+    ui_row(
+      title = "Score Summary",
+      DT::dataTableOutput(ns("pe_ProjectSummary")),
+      status = "info",
+      solidHeader = TRUE,
+      collapsible = TRUE
+      ),
+    ui_row(
+          inputId = ns("tabs"),
+          # title = "Client Detail",
+          tabPanel(
+            "Exits to Permanent Housing",
+            DT::dataTableOutput(ns("pe_ExitsToPH"))
+          ),
+          tabPanel(
+            "Benefits & Health Insurance at Exit",
+            DT::dataTableOutput(ns("pe_BenefitsAtExit"))
+          ),
+          tabPanel(
+            "Living Situation at Entry",
+            DT::dataTableOutput(ns("pe_LivingSituationAtEntry"))
+          ),
+          tabPanel(
+            "No Income at Entry",
+            DT::dataTableOutput(ns("pe_NoIncomeAtEntry"))
+          ),
+          tabPanel("Length of Stay",
+                   DT::dataTableOutput(ns("pe_LengthOfStay"))
+          ),
+          tabPanel(
+            "Median Homeless History Index",
+            DT::dataTableOutput(ns("pe_MedianHHI"))
+          ),
+          tabPanel(
+            "Long Term Homeless",
+            DT::dataTableOutput(ns("pe_LongTermHomeless"))
+          ),
+          tabPanel(
+            "VISPDAT Score Completion",
+            DT::dataTableOutput(ns("pe_ScoredAtPHEntry"))
+          ),
+    )
+  )
+}
+    
+#' body_coc_competition Server Functions
+#'
+#' @noRd 
+mod_body_coc_competition_server <- function(id){
+  moduleServer( id, function(input, output, session){
+    ns <- session$ns
+    
+    output$pe_ProjectSummary <-
+      DT::renderDataTable({
+        ptc <- pe_summary_final_scoring() |>
+          dplyr::filter(AltProjectName == input$pe_provider) |>
+          dplyr::pull(ProjectType)
+
+        pe_summary_final_scoring <- pe_summary_final_scoring()  |> 
+          dplyr::mutate_all(function(x) gsub("/", "÷", x))
```

Comment:
NIce job on the iteration! However `mutate_all` is superseded and thus may be removed in future versions of `dplyr`. You'll want to use `dplyr::mutate(dplyr::across` pattern here. See `?dplyr::across`

Reply:
Clicking `F1` on a function is one of the most helpful shortcuts because it pulls up the documentation for any function you're using.